HW_16/TODO/add.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+// TODO/add.php
+
+declare(strict_types=1);
+
+error_reporting(E_ALL);
+ini_set('display_errors', '1');
+
+require __DIR__ . '/header.php';
+require_once __DIR__ . '/db.php';
+
+if(session_status() != 'PHP_SESSION_ACTIVE'){
+    session_start();
+}
+$conn = getConnection();
+
+if ($_SERVER['REQUEST_METHOD'] === 'POST') {
+    $name = mysqli_real_escape_string($conn, $_POST['name']);
+    $userId = $_SESSION['current_user']['id'];
+
+    $sql = "INSERT INTO lists (name, id_users) VALUES ('$name', '$userId')";
```

Comment:
вторичный ключ, лучше называть <название таблицы в ед числе>_<название первичного ключа>
```suggestion
    $sql = "INSERT INTO lists (name, user_id) VALUES ('$name', '$userId')";
```